Application.hs

```diff
@@ -93,22 +93,12 @@ makeFoundation appSettings = do
 
   where
     getOAuthKeys :: String -> IO OAuthKeys
-    getOAuthKeys plugin = do
-        envClientId     <- lookupEnv $ plugin <> "_OAUTH_CLIENT_ID"
-        envClientSecret <- lookupEnv $ plugin <> "_OAUTH_CLIENT_SECRET"
-
-        case (envClientId, envClientSecret) of
-            (Just clientId, Just clientSecret) ->
-                return OAuthKeys
-                    { oauthKeysClientId     = pack clientId
-                    , oauthKeysClientSecret = pack clientSecret
-                    }
-
-            _ -> error
-                    $  plugin
-                    <> "_OAUTH_CLIENT_ID or "
-                    <> plugin
-                    <> "_OAUTH_CLIENT_SECRET not set"
+    getOAuthKeys plugin = OAuthKeys
```

Comment:
> What is "plugin"?

A Yesod `AuthPlugin` name. In this case it's assumed to be pre-capitalized and used as a prefix for environment variables.

> What's an example value

At the call site, `"GITHUB"` is used.

> Is there a better name for it?

It is and has been fine with me so far. Do you have any suggestions now that you know what it is?